### PR TITLE
Explicitly mentions the useage of enhanced CPS in the AI prompt when use enhanced CPS is  toggled 

### LINKED ISSUE #1856: Describe ECPS in prompt when used
Something like:
>This analysis uses PolicyEngine's Enhanced CPS, constructed from the 2022 Current Population Survey and the 2015 IRS Public Use File, and calibrated to tax, benefit, income, and demographic aggregates.

### PR DESCRIPTION
**Pull request by bharateshwq**
## Description
Fixes #1856

Adds Explicit descriptors to the AI prompt when enhanced CPS is turned on

## Changes

The code checks if enhanced CPS for USA is turned on and explicitly adds a descriptor to the AI prompt

### CODE REVIEW

**Comment by anth-volk** [comment]
@bharateshwq Could you run `make format` and push the code again?

**Comment by bharateshwq** [comment]
Hey @anth-volk Ive ran make format and pushed the code again

**Comment by anth-volk** [comment]
@bharateshwq Perfect! Thanks, will review later today.

File: src/pages/policy/output/promptContent.js [hunk 1]
```diff
@@ -10,11 +10,17 @@ export const promptContent = (
...
+  const isEnhancedCPS = regionKeyToLabel[region].includes("enhanced CPS");
+
   const policyDetails = `I'm using PolicyEngine, a free, open source tool to compute the impact of public policy. I'm writing up an economic analysis of a hypothetical tax-benefit policy reform. Please write the analysis for me using the details below, in their order. You should:
     
     * First explain each provision of the reform, noting that it's hypothetical and won't represents policy reforms for ${timePeriod} and ${
       regionKeyToLabel[region]
     }. Explain how the parameters are changing from the baseline to the reform values using the given data.
+     ${
+       isEnhancedCPS &&
+       "*  Explicitly mention that This analysis uses PolicyEngine's Enhanced CPS, constructed from the 2022 Current Population Survey and the 2015 IRS Public Use File, and calibrated to tax, benefit, income, and demographic aggregates."
```
Comment (anth-volk) [inline src/pages/policy/output/promptContent.js:22]:
```suggestion
       "*  Explicitly mention that this analysis uses PolicyEngine's Enhanced CPS, constructed from the 2022 Current Population Survey and the 2015 IRS Public Use File, and calibrated to tax, benefit, income, and demographic aggregates."
```

Just a minor change for grammatical reasons

**Review by anth-volk — CHANGES_REQUESTED** [review]
Thanks so much for your contribution! Just one minor change, and then this should be good to merge.

**Comment by anth-volk** [comment]
@bharateshwq Looks like you also merged your virtual environment. Could you either delete that, move it to a new folder, or rename it to `venv` and add `venv/*` to `.gitignore`? You'll also have to run `git reset venv` to remove it from git tracking.

**Comment by bharateshwq** [comment]
hey @anth-volk ive deleted them now,
thank you for being patient

**Comment by anth-volk** [comment]
@bharateshwq I'm sorry, this still contains the capitalized "This" that was an issue before

**Review by anth-volk — APPROVED** [review]
@bharateshwq Thanks for your contribution to this! All looks good.

**Review by AKANKSHBC — APPROVED** [review]

**Comment by anth-volk** [comment]
Hi @AKANKSHBC. While we are an open-source project open to contributors from all backgrounds, we typically prefer to work one-on-one with contributors before granting them the ability to approve PRs. I'm not sure how you were able to grant approval, to be honest. If you'd like to contribute in a meaningful way, whether it be writing new features or anything else, please respond here indicating your interest and we can work out a way to connect. Otherwise, please avoid approving PRs and doing any other administrative tasks moving forward.

**Pull request merged**